ldap2pg/script.py

```diff
@@ -33,8 +34,15 @@ def wrapped_main():
     config = Configuration()
     config.load()
 
-    ldapconn = create_ldap_connection(**config['ldap'])
-    pgconn = create_pg_connection(dsn=config['postgres']['dsn'])
+    try:
+        ldapconn = create_ldap_connection(**config['ldap'])
+        pgconn = create_pg_connection(dsn=config['postgres']['dsn'])
+    except ldap3.core.exceptions.LDAPExceptionError as e:
+        message = "Failed to connect to LDAP: %s" % (e,)
+        raise UserError(message)
```

Comment:
@julmon @pgiraud here, i carefully raise UserError at the *same* level of code which *know* that options comes from user (`config[...]`).